Kindly Try with this changes.

Dark1z opened the pull request:
Kindly Try with this changes , to fix `Pixel V1` Partition Detection.   

1. Added all partition in `twrp.fstab` located at `rootdir/recovery/twrp.fstab`.   
2. Added USB-OTG detection in `fstab.sprout` at `rootdir/root/fstab.sprout`.   
3. Added this line `$(LOCAL_PATH)/rootdir/recovery/twrp.fstab:root/etc/twrp.fstab` for `PRODUCT_COPY_FILES` in `seedmtk.mk`  by referring on XDA here : [[DEV]How to compile TWRP touch recovery](https://forum.xda-developers.com/showthread.php?t=1943625).   

If anyone can build it , then I can test it on my device.
(Just do not know how to build from Source on Windows)
@Dees-Troy , @bigbiff  & of-course @MSF-Jarvis Kindly take a note.

I have tested by Un-Packing the `.img` file and then Re-Pack it and flash.
Works Flawlessly , Including `USB OTG`. 😄 .

Best Regards.

Commit:
Try with this changes.

Dark1z commented on rootdir/recovery/twrp.fstab:
```diff
@@ -1,10 +1,21 @@
+/data           ext4     /dev/block/platform/mtk-msdc.0/by-name/userdata
+/cache          ext4     /dev/block/platform/mtk-msdc.0/by-name/cache
+/external_sd    vfat     /dev/block/mmcblk1p1      /dev/block/mmcblk1             flags=display="External SDCard";storage;wipeingui;removable
+/usb_otg        vfat     /dev/block/sda1           /dev/block/sda                 flags=display="USB OTG";storage;wipeingui;removable
```
USB-OTG

Dark1z commented on rootdir/root/fstab.sprout:
```diff
@@ -13,3 +13,4 @@
 /dev/block/platform/mtk-msdc.0/by-name/oem       /oem        ext4       ro,context=u:object_r:oemfs:s0,nosuid,nodev                 wait
 /devices/platform/mtk-msdc.1/mmc_host*           auto        auto       defaults                                                    voldmanaged=sdcard0:auto,encryptable=userdata,noemulatedsd
 /dev/block/platform/mtk-msdc.0/by-name/frp       /persistent emmc       defaults                                                    defaults
+/devices/platform/mt_usb*                        auto        vfat       defaults                                                    voldmanaged=usbotg:auto,encryptable=userdata
```
USB-OTG.
Copied from `Boot`.

Dark1z commented on seedmtk.mk:
```diff
@@ -22,5 +22,6 @@ $(call inherit-product-if-exists, device/mediatek/sprout-common/sprout.mk)
-    $(LOCAL_PATH)/rootdir/root/kernel:kernel
+    $(LOCAL_PATH)/rootdir/root/fstab.sprout:root/fstab.sprout \
+    $(LOCAL_PATH)/rootdir/root/kernel:kernel \
+    $(LOCAL_PATH)/rootdir/recovery/twrp.fstab:root/etc/twrp.fstab
```
By Referring on XDA here : [[DEV]How to compile TWRP touch recovery](https://forum.xda-developers.com/showthread.php?t=1943625).

msfjarvis commented:
Cool, I'll push it to gerrit when I get back home. Can you reach out to me on Telegram so I can pass you a test build? I've cracked the screen on my PixelV1, plus I'm not home atm so can't really test with it right now. I'm [MSFJarvis](https://t.me/MSFJarvis) on Telegram.

Dark1z commented:
I am Not on Telegram. (WhatsApp & Hike already eat my Brains)
Can you please pass me the test build on XDA , Here : [[TWRP][OFFICIAL] TWRP for LAVA Pixelv1](https://forum.xda-developers.com/lava-pixel-v1/orig-development/recovery-twrp-t3353364).
I go by [Dark❶](https://forum.xda-developers.com/member.php?u=7292542) .

Regards.

Commit:
Correction.

Forgot `recovery/` in `$(LOCAL_PATH)/rootdir/recovery/twrp.fstab:recovery/root/etc/twrp.fstab`

Dark1z commented on seedmtk.mk:
```diff
@@ -22,5 +22,6 @@ $(call inherit-product-if-exists, device/mediatek/sprout-common/sprout.mk)
-    $(LOCAL_PATH)/rootdir/root/kernel:kernel
+    $(LOCAL_PATH)/rootdir/root/fstab.sprout:root/fstab.sprout \
+    $(LOCAL_PATH)/rootdir/root/kernel:kernel \
+    $(LOCAL_PATH)/rootdir/recovery/twrp.fstab:recovery/root/etc/twrp.fstab
```
By Referring on XDA here : [[DEV]How to compile TWRP touch recovery](https://forum.xda-developers.com/showthread.php?t=1943625).

Corrected.
Forgot to add `recovery/` after `:` in this line.

Dark1z commented:
@MSF-Jarvis , Sir ,
Have you build the test build yet???

msfjarvis commented:
Not home dude, chill your tits and wait for me to post it on XDA.

Dark1z commented:
@MSF-Jarvis ,
:rofl: 

Out station ??? :confused: :worried:

Seriously DuDe !!! u had to put there "TiTs" . :stuck_out_tongue_winking_eye:

Dark1z commented:
@MSF-Jarvis ,
Kindly see the results here : https://forum.xda-developers.com/showpost.php?p=72652398&postcount=39

msfjarvis commented:
Don't ping me everywhere if you really want me to work on this. Stick to XDA, posting here sends unnecessary notifications to Dees, bigbiff and other members of the org.

Dark1z commented:
Opps! Sorry. Kinda new on GitHub.
This is the last one.
Unless utterly necessary, I'll be posting here.

Sincere apologies.

Dark1z commented:
Any Updates ???.

Dark1z commented:
I think it's high time to close this...
